src/VirtualClient/VirtualClient.Monitors/AmdSmiMonitor.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Monitors
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO.Abstractions;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using global::VirtualClient;
+    using global::VirtualClient.Contracts;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Telemetry;
+
+    /// <summary>
+    /// The Performance Counter Monitor for Virtual Client
+    /// </summary>
+    public class AmdSmiMonitor : VirtualClientIntervalBasedMonitor

```

Comment:
Could you make a MONITOR profile for this so that people understand the dependency? Like can you install it with apt/dnf or chocolatey? or needs to be downloaded via blob?